From: intrigeri <intrigeri@boum.org>
Date: Wed, 17 Aug 2011 17:13:54 +0200
Subject: Actually install main programs.

---
 setup.py |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/setup.py b/setup.py
index 35d282e..08e4370 100644
--- a/setup.py
+++ b/setup.py
@@ -10,4 +10,5 @@ setup(name='MAT',
       license='GPLv2',
       url='https://gitweb.torproject.org/user/jvoisin/mat.git',
       packages=['mat'],
+      scripts= [ 'mat-cli.py', 'mat-gui.py' ],
      )
-- 
