From: intrigeri <intrigeri@boum.org>
Date: Sun, 23 Oct 2011 15:00:53 +0200
Subject: Do not try to install files that we don't ship anymore.

---
 setup.py |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/setup.py b/setup.py
index 2332505..8c7d44d 100755
--- a/setup.py
+++ b/setup.py
@@ -9,7 +9,7 @@ setup(name='MAT',
       author_email='julien.voisin@dustri.org',
       license='GPLv2',
       url='https://gitweb.torproject.org/user/jvoisin/mat.git',
-      packages=['mat', 'mat.hachoir_editor', 'mat.pdfrw', 'mat.bencode',
+      packages=['mat', 'mat.hachoir_editor', 'mat.bencode',
           'mat.tarfile'],
       scripts=['mat-cli', 'mat-gui'],
       data_files=[
-- 
