From: intrigeri <intrigeri@boum.org>
Date: Thu, 18 Aug 2011 11:10:32 +0200
Subject: Fix TryExec program name in .desktop file.

---
 mat.desktop |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/mat.desktop b/mat.desktop
index e487359..fa0a164 100644
--- a/mat.desktop
+++ b/mat.desktop
@@ -6,6 +6,6 @@ Version=1.0
 Type=Application
 Comment=Metadata Anonymisation Toolkit
 Comment[Fr]=Kit d'anonymisation de metadata
-TryExec=mat-gui.py
+TryExec=mat-gui
 Terminal=false
 Categories=Utility;FileTools;Security
-- 
