From: jvoisin <julien.voisin@dustri.org>
Date: Fri, 20 Jan 2012 18:37:11 +0100
Subject: Fix a critical bug (thanks to omaylorga)

---
 mat-gui |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/mat-gui b/mat-gui
index 380e4d8..244c028 100755
--- a/mat-gui
+++ b/mat-gui
@@ -533,7 +533,7 @@ non-anonymised) file to output archive'))
             if self.liststore[line][3] != _('Clean (strict)'):
                 # if the file is not already strict cleaned
                 if self.force or not self.liststore[line][0].file.is_clean():
-                    if self.liststore[line][0].file.remove_all_ugly():
+                    if self.liststore[line][0].file.remove_all_strict():
                         # if everything went fine
                         self.liststore[line][3] = _('Clean (strict)')
                 if self.backup:  # the backup copy state
-- 
