From: intrigeri <intrigeri@boum.org>
Date: Thu, 18 Aug 2011 11:11:07 +0200
Subject: Fix French comment translation in .desktop file.

---
 mat.desktop |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/mat.desktop b/mat.desktop
index fa0a164..b457e5e 100644
--- a/mat.desktop
+++ b/mat.desktop
@@ -5,7 +5,7 @@ GenericName=mat
 Version=1.0
 Type=Application
 Comment=Metadata Anonymisation Toolkit
-Comment[Fr]=Kit d'anonymisation de metadata
+Comment[Fr]=Outil d'anonymisation de metadonnées
 TryExec=mat-gui
 Terminal=false
 Categories=Utility;FileTools;Security
-- 
