From: intrigeri <intrigeri@boum.org>
Date: Thu, 25 Aug 2011 00:29:16 +0200
Subject: Fix ugly typo.

---
 mat-gui |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/mat-gui b/mat-gui
index a73fc40..d92df74 100755
--- a/mat-gui
+++ b/mat-gui
@@ -471,7 +471,7 @@ non-anonymised) file to output archive'))
                     self.liststore[line][0].file.remove_all()
                 if self.backup:  # the backup copy state
                     self.liststore[line][4] = self.liststore[line][0].file.output
-                self.liststore[line][3] = _('Clean(lossless)')
+                self.liststore[line][3] = _('Clean (lossless)')
             yield True
         self.statusbar.push(0, _('Ready'))
         yield False
-- 
